eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Pool.hs

```diff
@@ -196,7 +202,7 @@ poolCertTransition (PoolEnv slot pp) ps@PState {psStakePoolParams, psFutureStake
         tellEvent $ RegisterPool ppId
         pure $
           payPoolDeposit ppId pp $
-            ps {psStakePoolParams = eval (psStakePoolParams ∪ singleton ppId poolParams)}
+            ps {psStakePoolParams = eval (psStakePoolParams ⨃ singleton ppId poolParams)}
```

Comment:
nice!

Reply:
hmm, actually looking at it again, I don't think it changes anything, since we're making sure that the `ppId` isn't in the map before we add the pool params.

Reply:
Yes, I agree it makes no difference, though considering just the single-line, the newer version would be the more appropriate operation, as it reflects the intention, irrespective of the pre-condition. I'm happy to revert this change back unless you are fine with it :sweat_smile: